modules/mod-deep-learning/ExploreHuggingFaceDialog.h

```diff
@@ -0,0 +1,24 @@
+/**********************************************************************
+
+   Audacity: A Digital Audio Editor
+   Audacity(R) is copyright (c) 1999-2021 Audacity Team.
+
+   ExploreHuggingFaceDialog.h
+   Hugo Flores Garcia
+
+******************************************************************/
+/**
+
+\class ExploreHuggingFaceDialog
+\brief ExploreHuggingFaceDialog TODO
```

Comment:
Better to use the `@file` (or `\file`, no difference) command to name and briefly describe a file's purpose.

I prefer not to use the `\class` command at all, but instead just prefix the Doxygen comments to the class definition itself.

Reply:
resolved in 0d3f97f48

Reply:
resolved in 0d3f97f48